src/lib/customTraceType.js

```diff
@@ -6,6 +6,8 @@ export function plotlyTraceToCustomTrace(trace) {
     )
   ) {
     return 'area';
+  } else if (trace.type === 'timeseries') {
+    return 'timeseries';
   } else if (
```

Comment:
this is a hack, because even with autobinx and autobiny, I'm having trouble differentiating between line traces and timeseries, when I'm switching away from timeseries to line traces. 

Since autobinx and autobiny have been set when we switched to the timeseries trace, and are not erased anywhere, the if conditional recognizes a timeseries trace, even if the intention was to draw a line trace.

This works because when plotly.js doesn't recognize a trace type, it sets it to scatter. Maybe I could do the same hack with other custom scatter types. Seems like some confusions might happen when switching multiple times between different custom scatter types.